package.json

```diff
@@ -37,6 +37,16 @@
     ]
   },
   "devDependencies": {
-    "typescript": "^5.8.2"
+    "@cypress/react": "^9.0.0",
```

Comment:
Let's add the script to run test with coverage, that we will be able to see coverage.

Reply:
I was able to run cypress
![image](https://github.com/user-attachments/assets/9bc87cd2-a8e2-4d70-8569-a46e550919cf)

and just react scripts test, but I didn't see coverage. When I tried to run jest I have an error
![image](https://github.com/user-attachments/assets/0a96ed35-f5fc-4575-93e6-b131cf282100)


Try to add script for jest. Create jest config, elaborate to this topic to gain more knowledge.

Reply:
Adjusted, thx